components/LinkGrid/src/index.scss

```diff
@@ -0,0 +1,183 @@
+.denhaag-link-grid {
+  position: relative;
```

Comment:
Eigenlijk de bedoeling dat we alles wat mogelijk is en specifiek voor deze component is, in tokens te stoppen:

```
.denhaag-link-grid {
  position: var(--denhaag-link-grid-position);
}
```

Reply:
Done in de ge-update PR! Alleen nog niet voor alle classes.. moet ik de rest ook nog aanpassen allemaal?